modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -60,6 +60,8 @@ def submit_claim
 
         clear_saved_form(params[:form_id]) if params[:form_id]
 
+        send_confirmation_email(params[:education_benefit]) if response.ok?
```

Comment:
I'm thinking it might make sense to split this confirmation email logic out into its own sidekiq job. The reason is because we are adding an additional network call (in `#confirmation_email_template_id`) and I would rather not slow down (or cause errors) in getting the confirmation page response to the user.

@tsr-rise8 thoughts?

Reply:
@nathanbwright I didn't think it would be an issue having that extra call in there but we can certainly move it if you think it may be an issue.

Reply:
Yeah, I think it would it would be safer in this situation.